howso/engine/trainee.py

```diff
@@ -3778,6 +3695,7 @@ def _create(
                     max_wait_time=max_wait_time,
                     overwrite_trainee=overwrite,
                     resources=resources,
+                    trainee_id=trainee_id
```

Comment:
Need to cleanup trainee_id in this file as its no longer a parameter to create_trainee